plugins/inputs/cloudwatch/README.md

```diff
@@ -68,12 +68,20 @@ API endpoint. In the following order the plugin will attempt to authenticate.
   ## See http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html
   ratelimit = 200
 
+  ## Namespace-wide statistic filters (only gets used if no metrics are defined). Optional.
+  # statistic_exclude = [ "average", "sum", min", "max", sample_count" ]
+  # statistic_include = [ "average", "sum", min", "max", sample_count" ]
```

Comment:
I think `min` and `max` need to be `minimum` and `maximum` here in certain use cases? noticed `max` doesn't return under the `AWS/ECS` namespace if I specify it (silently omits), whereas `maximum` works fine

skimming the code below, possibly just a docs bug?